docs/setup/xml-config.md

```diff
@@ -23,8 +23,21 @@ Fast-DDS allows to specify DDS configuration through an XML file.
 In order to apply a configuration, the path to the XML file must be provided through the following environment variable:
 
 ```sh
-export FASTRTPS_DEFAULT_PROFILES_FILE=/path/to/the/xml/profile
+export FASTRTPS_DEFAULT_PROFILES_FILE='/path/to/the/xml/profile'
 ```
+Then run 
```

Comment:
Lines 28-40 are generally true of any environment variable added (though why is the restart necessary? At worst, you should only have to restart the daemon?). I could see adding a note to this effect at the top, but not in particular on this one DDS.

Reply:
Makes sense. Restarting your machine is just something we have had to do every time to disable the mulitcast. I have no idea why it's needed for our VMs. Also makes sense though if you don't want to specifically direct this page to VMs

Reply:
When a DDS is reconfigured, it's necessary to restart the daemon with a `ros2 daemon stop` . You can learn more about this at https://answers.ros.org/question/327348/what-is-ros2-daemon/ . It might be good to help explain this on this page, as well.

Reply:
I agree with Steve.
This could be an helpful note to add at the top.

However, why using `sudo` to edit the bashrc file?